Tests/varLib/varLib_test.py

```diff
@@ -442,6 +442,20 @@ def test_varlib_build_sparse_masters_MVAR(self):
         mvar_tags = [vr.ValueTag for vr in varfont["MVAR"].table.ValueRecord]
         assert all(tag in mvar_tags for tag in fontTools.varLib.mvar.MVAR_ENTRIES)
 
+    def test_varlib_build_VVAR_CFF2(self):
+        ds_path = self.get_test_input('TestVVAR.designspace')
+        suffix = '.otf'
+        expected_ttx_name = 'TestVVAR'
+        tables = ["VVAR"]
+
+        finder = lambda s: s.replace('.ufo', suffix)
+        varfont, model, _ = build(ds_path, finder)
```

Comment:
You know that the varLib.build function can also build from designspace file that has sources’s filenames pointing to TTX files? I think it’d be better to check in the repo only text files instead of binary OTF files as the test data files (unless the binary file is the only means of storing a piece of data).

Reply:
All your points make sense. Will fix.

Reply:
My apologies for the mixed case style in one line - that is ugly. The standard I see in fontTools is:
- new modules should follow PEP8
- new functions added to an existing module should follow the module's predominating style. (for example, varLib/__init__.py has some functions with snake-case, and but has mostly camel-case code, and some examples of both in the same function).
- new logic added to a function should follow the function style.